src/components/Slide/SlideBase/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { Slide as MaterialSlide, SlideProps as MaterialSlideProps } from "@material-ui/core";
+
+export interface SlideBaseProps extends MaterialSlideProps {
+    unmountOnExit?: boolean;
+    mountOnEnter?: boolean;
+}
+
+const SlideBase = React.forwardRef<unknown, SlideBaseProps>((props: SlideBaseProps, ref) => {
```

Comment:
Why is this unknown @@

Reply:
Because the ref attribute of Slide component is defined with unknown type in MuiV5. Therefore, I afraid that if I put some specific type instead of unknown, there will be some bugs when using with other components.
![image](https://user-images.githubusercontent.com/59761256/148544013-7c7caccc-d9ef-48cf-ad26-e5b2483185ea.png)